docs/sql/statements/copy-from.rst

```diff
@@ -287,6 +289,38 @@ Using the ``s3://`` scheme automatically sets the `shared`_ to true.
    0.51.x these connections are using the HTTPS protocol. Please make sure you
    update your firewall rules to allow outgoing connections on port ``443``.
 
+.. _sql-copy-from-azblob:
+
+``azblob``
+''''''''''
+
+You can use the ``azblob:///`` scheme to access files on the `Azure Blob Storage`_.
+
+URI must consist of scheme and absolute path to the resource.
+
+For example:
+
+.. code-block:: text
+
+    azblob:///dir1/dir2/file1.json
+
+All other parameters must be provided in the ``WITH`` clause.
```

Comment:
Should they not all be optional?

Reply:
`container` and `endpoint` are required

Reply:
Isn't endpoint almost always  blob.core.windows.net for azure blob storage?

Reply:
URI in CrateDB must start with scheme name (s3/file/azblob), that's kinda tech requirement to load proper plugin.

If we support full endpoint, it won't be `COPY TO https:...`, it will be `COPY TO azblob:... ` and we would have to introduce dedicated endpoint param anyway.

Reply:
Why? Parsing the URI to determine whether it’s a full endpoint or just a container name is straightforward, and this actually supports the idea of including the container name in the URI. The entire purpose of a URI is to uniquely identify a resource, which isn’t possible without specifying the container.

https://duckdb.org/docs/extensions/azure.html#for-azure-blob-storage